merfi/tests/test_iso.py

```diff
@@ -0,0 +1,21 @@
+import os
+import pytest
+import subprocess
+from merfi.iso import Iso
+
+class TestIso(object):
+
+    def setup(self):
+        self.iso = Iso([])
+
+    def test_sha256sum(self, tmpdir):
+        f = tmpdir.join('test.iso')
+        f.write('ISOCONTENTS')
+        self.iso.output = str(f)
+        self.iso.make_sha256sum()
+        checksum_file = self.iso.output + '.SHA256SUM'
+        with open(checksum_file, 'r') as shaf:
+            assert shaf.read() == "d8d322f6864229f8c9ef1b0845dd9e182c563c508fec30618fdb9b57c70a0147  test.iso\n"
+            # ... and validate checksum_file's syntax with sha256sum:
+            os.chdir(os.path.dirname(checksum_file))
+            assert subprocess.call(['sha256sum', '-c', checksum_file]) == 0
```

Comment:
would it be too much to ask not to do a subprocess call here? can we just do it with Python?